Formula/http-prompt.rb

```diff
@@ -0,0 +1,109 @@
+class HttpPrompt < Formula
+  include Language::Python::Virtualenv
+
+  desc "Interactive command-line HTTP client with autocomplete and syntax highlighting"
+  homepage "https://http-prompt.com"
+  url "https://files.pythonhosted.org/packages/bf/e2/bc5b0df107afcac65fde7015df48cbe9b4d877d1d0818203544ed1a41d4c/http-prompt-2.1.0.tar.gz"
+  sha256 "eee71a00fed0b8a2a35bb338b269be7a20e8a1a6f6465a65561d76a21521e7f3"
+  license "MIT"
+  head "https://github.com/httpie/http-prompt.git", branch: "master"
+
+  depends_on "python@3.10"
+  depends_on "six"
```

Comment:
we'll need to exclude `six` in `pypi_formula_mappings.json` (otherwise `brew update-python-resources http-prompt` will add it back)